RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -310,6 +323,7 @@ boolean L9_chasmBuild()
 	visit_url("place.php?whichplace=orc_chasm&action=bridge"+(to_int(get_property("chasmBridgeProgress"))));
 
 	// use any keepsake boxes we have
+
```

Comment:
ah you must have been thinking about putting it here already. This is the spot attempting to pull should go